tensorflow/python/ops/nn_impl.py

```diff
@@ -1394,7 +1394,7 @@ def batch_normalization(x,
       normalized over (the 'depth' dimension(s)), and dimension 1 for the
       others which are being normalized over.
       `mean` and `variance` in this case would typically be the outputs of
-      `tf.nn.moments(..., keep_dims=True)` during training, or running averages
+      `tf.nn.moments(..., keepdims=True)` during training, or running averages
```

Comment:
This is `keepdims` in `master`, thank you for fixing the documentation to have this correct.

https://github.com/tensorflow/tensorflow/blob/8569d12747c13ac16e864db1482b6754904371c8/tensorflow/python/ops/nn_impl.py#L1241